src/handlebars/faq.handlebars

```diff
@@ -0,0 +1,187 @@
+    <DD> Yes! We provde downloads of <a href="https://adoptopenjdk.net/jmc.html">JMC</a> which are built through
+         <a href="https://ci.adoptopenjdk.net/view/JMC/job/jmc-build/">our CI</a>
+    <DT> <strong>What is the difference between HotSpot and OpenJ9 downloads?</strong>
+    <DD> HotSpot is the default Java Virtual Machine that is included in the OpenJDK codebase. OpenJ9 is an
+         alternative Java Virtual Machine developed by IBM for their own proprietory JVM and was open-sourced in
+         2017. You may find that OpenJ9 uses less memory than the HotSpot variants. Have a look at the
+         <a href="https://www.eclipse.org/openj9/oj9_faq.html">OpenJ9 FAQ</a> or the
+         <a href="https://www.eclipse.org/openj9/docs/cmdline_migration/">transition document</a> for more information
+    <DT> <strong> What are the OpenJ9 "Large Heap" variants?</strong>
+    <DD> The Large Heap variants of the OpenJ9 builds (also known as the "non-compressed references builds) allow
+         for java heap sizes greater than 57Gb. If you need heap sizes that large, then pick the large heap versions
+         (they have XL in the download filenames). The large heap versions are not on all platforms but the number
+         have increased over time.
+    <DT> <strong>I'm worried about security vulnerabilities in your project compared to other openjdk implementations</strong>
+    <DD> the AdoptOpenJDK project does not work on security vulnerability updates independently of the community
+         code base held at OpenJDK. For bugs including security issues, we work with the OpenJDK project to fix
+         bugs there, and then build, test and distribute the code here. The OpenJDK LTS updates projects are
+         working on vulnerabilities that are discussed amongst members of the OpenJDK Vulnerability Group.
+         Patches to fix those vulnerabilities are applied to the appropriate OpenJDK repository ‘upstream’, and
+         then we build, test and distribute the resulting binaries at AdoptOpenJDK. We do not compete with openjdk,
+         we work together with them and address security vulnerabilities together.
+    <DT> <strong>What are the CPU and PSU updates and which do you build?</strong>
+    <DD> AdoptOpenJDK ships binaries based on the PSU (Patch Set Update) release for the quarterly updates. This
+         contains additional fixes from the CPU (Critical Patch Update) and due to the very high levels of testing
+         which the AdoptOpenJDK projects runs on our release builds we are confident enough that the additional
+         fixes in the PSU updates will not cause any problems.
+    <DT> <strong>How can I get commercial support for AdoptOpenJDK binaries</strong>
+    <DD> The AdoptOpenJDK project does not itself provide commercial support for the binaries we produce,
+         however if you find and issue please raise it in the openjdk-support respository. If you need a stronger
```

Comment:
link to https://github.com/AdoptOpenJDK/openjdk-support

Reply:
Done - thanks